### PR TITLE
add reddit icon to header and footer

### PR DESCRIPTION
**Pull request by varun-tiwari**
added reddit icon to header and  footer

### CODE REVIEW

File: components/Header/Header.js [hunk 1]
```diff
@@ -20,6 +20,12 @@ const Header = () => {
           rel="noreferrer noopener">
           <FaDiscord className="w-6 h-6 sm:w-8 sm:h-8 hover:text-discord-200 text-discord-100 transition duration-300 ease-in-out" />
         </a>
+        <a
+          href="https://www.reddit.com/r/iamfreeCodeCamp/"
+          target="_blank"
+          rel="noreferrer noopener">
+          <FaReddit className="w-6 h-6 sm:w-8 sm:h-8 hover:text-reddit-200 text-reddit-100 transition duration-300 ease-in-out" />
```
Comment (evedes) [inline components/Header/Header.js:27]: I've seen you used text-reddit-100 and text-reddit-200. For this to work you need to add the reddit color in the tailwind.config.js. I'd suggest to use orange as the default color, so we need two shades of orange, one lighter and one darker, just like in the twitter case.

**Review by evedes — CHANGES_REQUESTED** [review]
Hey, good job with the reddit icon! 
Left only a small comment about using colors in tailwind.

Also, we're using yarn, so we don't need a package-lock.json in our codebase. Could you remove it? 

Thanks! And keep up the good work! ❤️

File: components/Header/Header.js (hunk 1 above)
Comment (evedes) [reply components/Header/Header.js:27]: Hey @varun-tiwari, are you going to finish this or can we move the ticket to someone else? 🙏🏻

**Comment by varun-tiwari** [comment]
@evedes  Hi Evedes, sorry for delay, could you please review my new commit,please let me know if any other changes neede.

**Comment by evedes** [comment]
Great job @varun-tiwari! ❤️  We're really thankful for your contribution!

**Comment by evedes** [comment]
@varun-tiwari your code is approved, but there are conflicts to solve. Reach out on Discord if you need help solving them! #contributors channel 👍🏻

**Comment by evedes** [comment]
I ended up giving you a hand mate! Good job anyway! ❤️ 
Merged here: https://github.com/iamfreeCodeCamp/iamfreecodecamp/pull/33
Closing the PR.

**Pull request closed without merging**